src/js/utils/cursor.js

```diff
@@ -98,87 +98,29 @@ export default class Cursor {
 
   get offsets() {
     const { sections } = this.post;
-    const selection = this.selection;
-    const { isCollapsed, rangeCount } = selection;
-    const range = rangeCount > 0 && selection.getRangeAt(0);
+    const { selection } = this;
 
-    if (!range) {
+    if (selection.rangeCount === 0 || !selection.getRangeAt(0)) {
```

Comment:
I think `selection.rangeCount === 0` is sufficient. I'm not sure what circumstance would result in the rangeCount being non-zero and `getRangeAt(0)` returning `undefined`